source/extensions/transport_sockets/tls/ssl_handshaker.h

```diff
@@ -137,6 +137,11 @@ class HandshakerFactoryImpl : public Ssl::HandshakerFactory {
     return Ssl::HandshakerCapabilities{};
   }
 
+  Ssl::SslCtxCb sslctxCb(Ssl::HandshakerFactoryContext&) const override {
+    // The default handshaker impl doesn't additionally modify SSL_CTX.
+    return [](SSL_CTX*) {};
```

Comment:
`return nullptr;`, otherwise you're always executing empty lambda.

Reply:
Done.